source/content/read-environment-config.md

```diff
@@ -110,57 +100,59 @@ The `$_SERVER` variable contains sensitive data about a site and should not be p
-      ["SERVER_PROTOCOL"]=>
-      string(8) "HTTP/1.1"
-      ["GATEWAY_INTERFACE"]=>
-      string(7) "CGI/1.1"
-      ["REMOTE_ADDR"]=>
-      string(12) "72.188.192.8"  <= NOT A SITE IP ADDRESS
-      ["REMOTE_PORT"]=>
-      string(5) "55982"
-      ["SERVER_ADDR"]=>
-      string(13) "10.223.192.37"  <= NOT A SITE IP ADDRESS
-      ["SERVER_PORT"]=>
-      string(5) "11846"
-      ["SERVER_NAME"]=>
-      string(31) "endpointe7208f3b.chios.panth.io"
-      ["REDIRECT_STATUS"]=>
-      string(3) "200"
-      ["PATH_TRANSLATED"]=>
-      string(51) "/srv/bindings/non-static-binding-string-inserted-here/code"
-      ["HTTPS"]=>
-      string(3) "OFF"
-      ["SCRIPT_FILENAME"]=>
-      string(62) "/srv/bindings/non-static-binding-string-inserted-here/code//index.php"
-      ["HTTP_HOST"]=>
-      string(25) "example-wp-site.pantheonsite.io"
-      ["HTTP_USER_AGENT"]=>
-      string(120) "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_10_5) AppleWebKit/537.36 (KHTML, like Gecko)         Chrome/46.0.2490.80 Safari/537.36"
-      ["HTTP_ACCEPT"]=>
-      string(74) "text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8"
-      ["HTTP_ACCEPT_LANGUAGE"]=>
-      string(14) "en-US,en;q=0.8"
-      etc...
-    }
+```php
+<?php var_dump($_SERVER);  ?>
+array(63) {
+    ["SERVER_SOFTWARE"]=>
+    string(11) "nginx/1.4.7"
+    ["REQUEST_URI"]=>
+    string(1) "/"
+    ["USER"]=>
+    string(32) "non-static-binding-string-inserted-here"
+    ["HOME"]=>
+    string(46) "/srv/bindings/non-static-binding-string-inserted-here"
+    ["SCRIPT_NAME"]=>
+    string(10) "/index.php"
+    ["DOCUMENT_URI"]=>
+    string(10) "/index.php"
+    ["DOCUMENT_ROOT"]=>
+    string(51) "/srv/bindings/non-static-binding-string-inserted-here/code"
+    ["SERVER_PROTOCOL"]=>
+    string(8) "HTTP/1.1"
+    ["GATEWAY_INTERFACE"]=>
+    string(7) "CGI/1.1"
+    ["REMOTE_ADDR"]=>
+    string(12) "72.188.192.8"  <= NOT A SITE IP ADDRESS
+    ["REMOTE_PORT"]=>
+    string(5) "55982"
+    ["SERVER_ADDR"]=>
+    string(13) "10.223.192.37"  <= NOT A SITE IP ADDRESS
+    ["SERVER_PORT"]=>
+    string(5) "11846"
+    ["SERVER_NAME"]=>
+    string(31) "endpointe7208f3b.chios.panth.io"
+    ["REDIRECT_STATUS"]=>
+    string(3) "200"
+    ["PATH_TRANSLATED"]=>
+    string(51) "/srv/bindings/non-static-binding-string-inserted-here/code"
+    ["HTTPS"]=>
+    string(3) "OFF"
+    ["SCRIPT_FILENAME"]=>
+    string(62) "/srv/bindings/non-static-binding-string-inserted-here/code//index.php"
+    ["HTTP_HOST"]=>
+    string(25) "example-wp-site.pantheonsite.io"
+    ["HTTP_USER_AGENT"]=>
+    string(120) "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_10_5) AppleWebKit/537.36 (KHTML, like Gecko)         Chrome/46.0.2490.80 Safari/537.36"
+    ["HTTP_ACCEPT"]=>
+    string(74) "text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8"
+    ["HTTP_ACCEPT_LANGUAGE"]=>
+    string(14) "en-US,en;q=0.8"
+    etc...
```

Comment:
this one has "etc" and the first doesn't? Do we want  them  to match for consistency?

Reply:
agreed. thank you!
fixed and marking as resolved